frontend/views/containers/payments/PaymentsMixin.js

```diff
@@ -80,8 +55,8 @@ const PaymentsMixin: Object = {
       }
       return payments
     },
-    async getHistoricalPaymentByHash (hash: string) {
-      const paymentsKey = `payments/${this.ourUsername}/${this.currentGroupId}`
+    async getHistoricalPaymentByHash (period: string, hash: string) {
```

Comment:
@Silver-IT Could we maybe also update the name of this method to something that tells us passing a payment period is now required to use it?
(There is a large number of payment-related Vuex getters, group contract getters and utility methods across the front-end code-base with _more or less_ similar names, so the more detailed and explicit the names are, the more helpful it is, when trying to determine which one to use to solve a particular problem. :) )

Reply:
That's reasonable. Will do that.